fix racy events test

jessfraz opened the pull request.

jfrazelle commented:
ping @LK4D4 @icecrime @estesp @duglin whoever gets here first it fixes the events test with jenkins see https://github.com/docker/docker/pull/10998#discussion_r25313316

tiborvass commented:
LGTM

jessfraz committed:
fix racy events test

Docker-DCO-1.1-Signed-off-by: Jessie Frazelle <jess@docker.com> (github: jfrazelle)

jfrazelle commented:
LGTM
